src/wizard/clojure/index.md

```diff
@@ -0,0 +1,56 @@
+---
+name: Clojure
+doc_link: https://docs.sentry.io/platforms/clojure/
```

Comment:
We should add some docs first before we merge this. Right now it 404's

Reply:
We'd need a `clojure` folder here with at least an `index.mdx`:

https://github.com/getsentry/sentry-docs/tree/master/src/platforms